__init__.py

```diff
@@ -0,0 +1,73 @@
+from openfecwebapp.local_config import port, debug, host
+from flask import Flask, render_template, request
+from openfecwebapp.views import (render_search_results, render_table,
+    render_page)
+from openfecwebapp.api_caller import (load_search_results,
+    load_single_type, load_totals)
+
+app = Flask(__name__)
+
+def _filter_empty_params(params):
+    new_params = {}
+    for p in params:
+        if params[p]:
+            new_params[p] = params[p]
+
+    return new_params
+
+def _add_fields_star(params):
+    # move from immutablemultidict -> multidict -> dict
+    params = params.copy().to_dict()
+    params = _filter_empty_params(params)
+    params['fields'] = '*'
+
+    return params
+
+@app.route('/')
+def home_page():
+    return render_template('search.html');
+
+@app.route('/search')
+def search():
+    query = request.args.get('search')
+    if query:
+        return render_search_results(load_search_results(query), query)
+    else:
+        return home_page()
```

Comment:
@noahmanger I'd like your input on this.

Right now, going to either http://site/ or http://site/search gets you the home page. The search disambiguation page is at /search?search=thing. 

There is duplication because I feel like its weird to send people to /search when they come into /, and weird to have the search disambiguation at /?search=thing.

Do you have a different idea? Thanks.